### DISCUSSION TITLE
How to disable ESR in order to run react-admin within flareact

### DISCUSSION THREAD

**Discussion by spartakb** [flareact/flareact#226] [Q&A]
I'm currently trying to follow this guide to run react-admin inside flareact (https://marmelab.com/blog/2022/02/02/bootstrap-your-react-admin-project-with-nextjs.html).    I realize this is a NextJS guide, but I'm hopping to be able to make the transplantation work in a similar fashion. 

In one of the steps it's requiring that the ssr (aka esr in flareact world) to be disabled via following approach:

```
import dynamic from "next/dynamic"
const App = dynamic(() => import("../src/admin/App"), { ssr: false })
```

How can I make a similar thing happen inside flareact.  To force a page not to render on the edge. 

Thank you in advance!

**Comment by traffisco** [comment]
Not supported.
Personally, I don't see any reason why they made it like that.

**Reply by spartakb** [reply]
Thanks, and very surprising. Such a great project, but not everything can render be ServerSide Rendered.
